intel_extension_for_transformers/neural_chat/models/base_model.py

```diff
@@ -389,7 +389,7 @@ def get_conv_template(self, model_path: str, task: str = "") -> Conversation:
         if self.conv_template:
             return
         if not task:
-            self.conv_template = PromptTemplate(self.get_default_conv_template(model_path).name)
+            self.conv_template = PromptTemplate(self.get_default_conv_template(model_path).name, clear_after_gen=True)

```

Comment:
is "skip_history" better?

Reply:
Afraid not. This is to align with old code:

https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/neural_chat/models/base_model.py#L396

https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/neural_chat/prompts/prompt.py#L205